ksql-engine/src/test/resources/query-validation-tests/project-filter.json

```diff
@@ -43,6 +43,29 @@
         }
       ]
     },
+    {
+      "name": "WHERE with many comparisons",
```

Comment:
Can you add the issue number, so we know why this test is important?

Reply:
Added the issue number.